### ISSUE TITLE
Data structure name in specification manager shall be the name of the PSM schema in the schema generator

### ISSUE THREAD

**Issue by martinnec** [opendata-mvcr/model-driven-data#117]
Here:

<img width="887" alt="image" src="https://user-images.githubusercontent.com/4235208/150200804-d57677c4-05d4-4ba7-82b5-9a9868194c92.png">

I would like to see:

<img width="884" alt="image" src="https://user-images.githubusercontent.com/4235208/150200736-0778d194-f6aa-4c38-92f0-fc3899b8e661.png">

**Comment by sstenchlak** [comment]
This works. The problem is that the specification manager prefers English over the Czech language, and in the schema generator, both Czech and English language mutations are created. You have probably edited only the Czech one; therefore, the change you have made is not visible in the specification manager.

Do you @martinnec want me to make some modifications to make it straightforward?

**Comment by martinnec** [comment]
Ok, then I would expect that for English mutation I do not see any label when there is no English label specified. In my "problem" screenshot, I see Czech labels in English mutation. It would be better to see no labels which forces me to complement them.

But let it be for now. I switch this to Discussion and assign @jakubklimek to give his opinion.

**Comment by jakubklimek** [comment]
OK, I did not know that we are generating both Czech and English versions of specifications (and schemas?) Makes sense though, especially when we want the solution to be independent of SSP and usable in other situations as well.

Nevertheless, the question here is about the behavior when there is no label, i.e.
> Ok, then I would expect that for English mutation I do not see any label when there is no English label specified. 

So, what is the behavior? Is there a fallback to the Czech label for English specifications (and vice versa) when English label is not specified? Will it be blank?

**Comment by sstenchlak** [comment]
Yes, there should be a fallback to the second preferred language (English or Czech) or to another language mutation that exists.

The intention is rather to show something than nothing.

I can visually distinguish labels that do not come from the current language mutation. For example: grey color and a badge with the label's language.

**Comment by jakubklimek** [comment]
> I can visually distinguish labels that do not come from the current language mutation. For example: grey color and a badge with the label's language.

@sstenchlak I would quite like this solution for the documentation. It makes it clear where the text comes from and the author can act on it. @martinnec what do you think?

**Comment by jakubklimek** [comment]
Even though stuctural editor is deprecated, in the new manager, this problem still exists for model names. Consistently with CME, I would still support @sstenchlak proposition, i.e. greyed out + language tag for foreign language.

**Comment by jakubklimek** [comment]
The language tag is still missing in the data structure tile

<img width="878" height="599" alt="Image" src="https://github.com/user-attachments/assets/c6f84245-c5c4-4f0a-81f7-61ff9eb10cb8" />
